openvdb_houdini/houdini/ParmFactory.cc

```diff
@@ -912,8 +913,19 @@ class OP_OperatorDW: public OP_Operator
             os << "= " << english << " =\n\n"
                 << "#type: node\n"
                 << "#context: " << flavorStr << "\n"
-                << "#internal: " << name << "\n\n"
-                << doc << "\n\n";
+                << "#internal: " << name << "\n";
+                if (isASWF) {
```

Comment:
This sort of logic belongs in an `OpPolicy` (in this case, `ASWFOpenVDBOpPolicy`), though how best to expose it is not clear.  One possibility would be to add `OpPolicy::docHeader` and `OpPolicy::docFooter` methods; another would be to add a `docTemplate` method, with some suitable syntax to interpolate the documentation body into the template.

Reply:
I think that's a good suggestion, I'll have a look at how to go about doing this. The header and footer implementation is probably simplest, so I think I'll start there.

Reply:
Done - just updated the PR.

Reply:
It's unintuitive at best that the "footer" is inserted in the middle of the page.  Either move it to the bottom or go with a document template instead.  I prefer the flexibility of the latter.

Reply:
I think you perhaps have a better idea of what you're after with this document template approach, Peter. Do you want to take this task over from me? I just want to try and avoid a bit of back-and-forth if possible.

Reply:
I can take a stab at it, yes.